mention experimental support for other mesh types in the README.md (#718)
trixi-framework/Trixi.jl: README.md

installation and postprocessing procedures. Its features include:

* Hierarchical quadtree/octree grid with adaptive mesh refinement

> * Experimental support of curvilinear structured and unstructured meshes
>   (cf. [overview in the documentation](https://trixi-framework.github.io/Trixi.jl/stable/overview/#Semidiscretizations))

Reviewer: ```suggestion
* Experimental support for curvilinear structured and unstructured meshes
  (see also the [overview in the documentation](https://trixi-framework.github.io/Trixi.jl/stable/overview/#Semidiscretizations))
```
`cf.` is IMHO only correctly used in the sense of "compare to/against", but in the sense of "see also".
